### PR TITLE
app-forensics/volatility3: add 2.4.1

### PR DESCRIPTION
**Pull request by hamarituc**

### CODE REVIEW

File: app-forensics/volatility3/volatility3-2.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE="crypt disasm jsonschema leechcore snappy test yara"
+
+RDEPEND="
+	>=dev-python/pefile-2017.8.1[${PYTHON_USEDEP}]
+	crypt? ( >=dev-python/pycryptodome-3[${PYTHON_USEDEP}] )
+	disasm? ( >=dev-libs/capstone-3.0.5[python,${PYTHON_USEDEP}] )
+	jsonschema? ( >=dev-python/jsonschema-2.3.0[${PYTHON_USEDEP}] )
+	leechcore? ( >=dev-python/leechcorepyc-2.4.0[${PYTHON_USEDEP}] )
+	snappy? ( app-arch/snappy:0/1 )
```
Comment (thesamesam) [inline app-forensics/volatility3/volatility3-2.4.1.ebuild:35]: Really? What's the story here?

(Both with the subslot pinning and also the switch from a Python dep.)

Comment (hamarituc) [reply app-forensics/volatility3/volatility3-2.4.1.ebuild:35]: Upstream decided to drop the python dependency in favor of a ctypes interface. See https://github.com/volatilityfoundation/volatility3/pull/913. The code now references to `libsnappy.so.1` directly so I thought a sublot pinning would be necessary.

Comment (thesamesam) [reply app-forensics/volatility3/volatility3-2.4.1.ebuild:35]: Sorry, I meant to come back and explain this bit.

Yeah, you're right, but the new snappy subslot in 1.1.10-r1 is for the same SONAME but to represent that they broke ABI to force a rebuild, so we need to do || ( ... ) and add a comment saying that the SONAME is hardcoded. 

Could you do that in a followup?

Comment (hamarituc) [reply app-forensics/volatility3/volatility3-2.4.1.ebuild:35]: see #30621

**Pull request closed without merging**